test/trento/application/event_handlers/stream_roll_up_event_handler_test.exs

```diff
@@ -17,6 +21,36 @@ defmodule Trento.StreamRollUpEventHandlerTest do
                         :max_stream_version
                       ])
 
+  test "should dispatch the host roll-up command" do
+    host_id = Faker.UUID.v4()
+
+    :ok =
+      Trento.EventStore.append_to_stream(
+        host_id,
+        0,
+        Enum.map(0..@max_stream_version, fn _ ->
+          event = TestEvent.new!(%{"data" => Faker.StarWars.quote()})
+
+          %EventStore.EventData{
+            causation_id: UUID.uuid4(),
+            correlation_id: UUID.uuid4(),
+            event_type: TypeProvider.to_string(event),
+            data: event,
+            metadata: %{}
+          }
+        end)
+      )
+
+    event = build(:host_registered_event, host_id: host_id)
```

Comment:
It would be strange to rollup in a host_registered_event, as it is the 1st event, but well, we get the point hehe